test/traffic_matcher.cc

```diff
@@ -47,6 +47,9 @@ namespace {
   using ots_matches_t = std::vector<ots_t>;
   using sid_t = baldr::GraphId;
   std::vector<std::pair<std::string, ots_matches_t> > test_cases {
+    //partial, partial with duplicate point
+    std::make_pair(R"({"trace":[{"lon":-76.376045,"lat":40.539207,"time":0},{"lon":-76.357056,"lat":40.541309,"time":99},{"lon":-76.357056,"lat":40.541309,"time":100}],"match_options":{"breakage_distance":10000}})",
```

Comment:
actually that wouldnt make this fail. the `max_route_distance` is the distance between the two input points not the projected points. noted this below